classes/RestRoutes.php

```
@@ -437,4 +445,66 @@ private static function processSendEmail(\WP_REST_Request $request): array
+     * Returns live send counters for a background send in progress.
+     *
+     * @param \WP_REST_Request $request REST request with 'campaignPostId'.
+     * @return array{sent:int, failed:int, skipped:int, unsubed:int, total:int, running:bool}
+     */
+    public static function backgroundProgress(\WP_REST_Request $request): array
+    {
+        $campaignPostId = (int) $request->get_param('campaignPostId');
+        $campaign       = Campaigns::getCampaignById($campaignPostId);
+
+        if (!$campaign) {
+            return ['error' => 'Not found'];
```

PHPStan (2.2.2)

[error] Method Mawiblah\RestRoutes::backgroundProgress() should return array{sent: int, failed: int, skipped: int, unsubed: int, total: int, running: bool} but returns array{error: 'Not found'}.
Array does not have offset 'sent'.

(return.type)